ext/openssl/xp_ssl.c

```diff
@@ -1204,7 +1204,7 @@ static RSA *php_openssl_tmp_rsa_cb(SSL *s, int is_export, int keylength)
 }
 #endif
 
-static int php_openssl_set_server_dh_param(php_stream * stream, SSL_CTX *ctx) /* {{{ */
+static zend_result php_openssl_set_server_dh_param(php_stream * stream, SSL_CTX *ctx) /* {{{ */
```

Comment:
nit: while at it, it might be worth adding then. return check where it s called.

Reply:
Added, thanks.

Reply:
Actually, the missing check would be nice to fix in stable versions... I'll make a PR just cherry-picking that fix.